set xmlns to many WEB-INF/web.xml files.

alexbodn opened the pull request:
lack of it gave parsing errors using openjdk-8.
i don't have other vendor's java.

Improved:
Implemented:
Documented:
Completed:
Reverted:
Fixed:
(OFBIZ-)

Explanation

Thanks:

alexbodn committed:
set xmlns to many WEB-INF/web.xml files.
lack of it gave parsing errors using openjdk-8.
i don't have other vendor's java.

alexbodn committed:
fixed a small glitch in the themes' style.css.
added missing map files to the minified openlayers files.
the name of the map file is being mentioned in the coresponding minified file.

JacquesLeRoux commented:
Hi Alex,

We had this already several times:
https://issues.apache.org/jira/browse/OFBIZ-6807
https://issues.apache.org/jira/browse/OFBIZ-11235
https://issues.apache.org/jira/browse/OFBIZ-11240
It should have been "fixed" (a workaround actually) with
https://issues.apache.org/jira/browse/OFBIZ-6993

In other words the obvious solution (the same than you propose) introduce another issue, see OFBIZ-6807

Please close if you agree

alexbodn committed:
Fixed: getEntityRefData service has a weird issue (OFBIZ-11401)

    It's reproductible on trunk demo though only visible in log, UI shows no issue

    R17 and R18 are also affected, R16 is OK.

    Ankush: This stacktrace should be coming for all entities where
    default-resource-name is defined on entity definition but no property exist in
    defined resource for key EntityDescription.entityname and FieldDescription.field.

    As we want to have these entity labels as much as possible, we can simply print
    warning about the missing entity descriptions and field descriptions for
    readability purpose instead of printing huge exceptions stacktrace.

    Thanks: Ankush Upadhyay and Pierre Smits for the help

alexbodn committed:
implemented ui label translation for the select unit of measure ajax.

alexbodn committed:
applied namespace to web.xml files

alexbodn committed:
unsafe-inline allowed in csp. needed for onclick like events in chrome.

alexbodn committed:
adopted upstream commits

alexbodn committed:
added div.row to allow multiple product summaries cards in a card container.

alexbodn committed:
added csp report-uri at /webtools/control/csp-report,
implemented to log csp reports from the browser as errors.

alexbodn committed:
one load of javascript for products

JacquesLeRoux commented:
I have done all it was possible from this code, closing here as it can't no more be merged
